zerver/models.py

```diff
@@ -1682,12 +1681,11 @@ def to_dict(self):
     @staticmethod
     def status_from_string(status):
         # type: (NonBinaryStr) -> Optional[int]
+        status_val = None
         if status == 'active':
             status_val = UserPresence.ACTIVE
         elif status == 'idle':
             status_val = UserPresence.IDLE
-        else:
-            status_val = None
```

Comment:
This should be part of an updated commit named 'mypy: Reorder and annotate variables around if statements.'

New commits incoming soon.